src/applications/appeals/testing/containers/App.jsx

```diff
@@ -82,6 +83,13 @@ export default function App(props) {
       <div className="row">
         <div className="usa-width-two-thirds medium-8 columns">{children}</div>
       </div>
+      {!noHeader.includes(currentPath) && (
```

Comment:
I'm reading this as if the header includes the current Path since it's a double negative not noHeader. Is that right?

Reply:
Yeah, it's a double negative. We want to include a footer on pages with a header